lib/bottom_navy_bar.dart

```diff
@@ -13,6 +13,8 @@ class BottomNavyBar extends StatelessWidget {
   BottomNavyBar({
     Key key,
     this.selectedIndex = 0,
+    this.itemSelectedWidth = 130,
+    this.itemWidth = 50,
```

Comment:
Hi @davidpatrickcoleman 
Thanks for this amazing contibution. Since this is a new feature could you please add tests to prevent this feature to break in the future? 
Thank you

Sorry for the later feedback!